mswindows/osgeo4w/package.sh

```diff
@@ -248,7 +241,9 @@ if [ -n "$PACKAGE_PATCH" ]; then
 
     # copy dependencies (TODO: to be reduced)
     cp -uv $DLLS apps/grass/grass$POSTFIX/bin
-    cp -uv /mingw64/etc/fonts/fonts.conf apps/grass/grass$POSTFIX/etc
+
+    # copy R batch files
```

Comment:
this should be checked whether we include these files already in another way, e.g. in the NSIS installer-

Reply:
Is this exact script really used for the NSIS installer? This script is used only by OSGeo4W (yet) from the best of my knowledge. I have some work prepared even before this PR was filed to use this script for our CI, to be really closer to what OSGeo4W packages, and could upload the artifacts that could be tested locally, and use ccache (that is already scripted here, we only have to download and upload the cache in CI). To not duplicate work, I was patiently waiting for this PR to be merged before continuing, but I would really want to continue to allow shaving some time off of the longest CI job that we have.

Reply:
No, NSIS does that here:
https://github.com/OSGeo/grass/blob/3e22b87722fb96c86c27f3e78405bc8c4fe0ed54/mswindows/GRASS-Packager.bat.tmpl#L65